unity-renderer/Assets/Scripts/MainScripts/DCL/WorldRuntime/ParcelScene.cs

```diff
@@ -95,6 +101,25 @@ public virtual void SetData(LoadParcelScenesMessage.UnityParcelScene data)
 
             OnSetData?.Invoke(data);
         }
+        
+        internal long EntityFromLegacyEntityString(string entityId)
+        {
+            switch (entityId)
+            {
+                case "0":
+                    return CONST_SCENE_ROOT_ENTITY ;
+                case "FirstPersonCameraEntityReference":
+                    return CONST_FIRST_PERSON_CAMERA_ENTITY_REFERENCE;
+                case "AvatarEntityReference":
+                    return CONST_AVATAR_ENTITY_REFERENCE;
+                case "AvatarPositionEntityReference":
+                    return CONST_AVATAR_POSITION_REFERENCE;
+                case "PlayerEntityReference":
+                    return CONST_THIRD_PERSON_CAMERA_ENTITY_REFERENCE;
+            }
+
+            return entityId.GetHashCode() << 9;
```

Comment:
A comment here saying that we are reserving the first 512 entities for special IDs wont hurt. Will also prevent accidentally changing the << 9